### ISSUE TITLE
Error fetching charge_mode data

### ISSUE THREAD

**Issue by flouck** [hacf-fr/hassRenaultZE#154]
## Configuration

All via HACS with the latest update.

## Describe the bug
I have a 2014 Renault Zoe, the charge mode entity is unavailable. Even after deactivating it, it still shows this error in the logs.

## Debug log
```
This error originated from a custom integration.

Logger: custom_components.renault.renault_vehicle
Source: helpers/update_coordinator.py:219
Integration: Renault (documentation, issues)
First occurred: 12:40:57 AM (1 occurrences)
Last logged: 12:40:57 AM

Error fetching VF1123441234 charge_mode data: This endpoint has been disabled: ('err.func.403', 'Access is denied for this resource')

```

**Comment by flouck** [comment]
I got this warning a bit after:
```
This error originated from a custom integration.

Logger: custom_components.renault.renault_vehicle
Source: custom_components/renault/renault_vehicle.py:130
Integration: Renault (documentation, issues)
First occurred: 12:40:57 AM (1 occurrences)
Last logged: 12:40:57 AM

`Access Denied` on HA coordinator charge_mode was not caught by `endpoint_available` method. It may be useful to open an issue on https://github.com/hacf-fr/renault-api/issues
```

**Comment by epenet** [comment]
⚠️ The Renault integration was originally merged into core in 2021.8 and finalised in 2021.10. The use of this custom component has been deprecated since then and should no longer be used

**Comment by epenet** [comment]
Note: you should raise an issue on https://github.com/hacf-fr/renault-api instead
